datalad/support/tests/test_gitrepo.py

```diff
@@ -881,3 +881,12 @@ def cmd_func(self, git_options=None, annex_options=None, options=None):
                         annex_options=to_options(JSON=True),
                         options=to_options(unused=True)),
         ['git', '-C/some/where', 'annex', '--JSON', 'my_cmd', '--unused'])
+
+
+@with_tempfile(mkdir=True)
+@with_tempfile()
+def test_GitRepo_count_objects(repo_path, dir_path):
+
+    repo = GitRepo(repo_path, create=True)
+    eq_(isinstance(repo.count_objects, dict), True)   # test if dictionary returned
+    eq_(len(repo.count_objects), 8)                   # test if dictionary is of expected size
```

Comment:
what about solidifying it to really get all the keys and values, since happen git renames one of them we better know, i.e. compare this dict to the actual dict `{'count': 0, 'size': 0, ...}`